src/bitmessageqt/addressvalidator.py

```diff
@@ -1,14 +1,29 @@
-from PyQt4 import QtGui
+"""
+Address validator module.
+"""
+# pylint: disable=too-many-branches,relative-import,too-many-arguments,unused-argument
+
 from Queue import Empty
+from PyQt4 import QtGui
+from utils import str_chan
+from account import getSortedAccounts
```

Comment:
Please follow PEP8. You mixed local and library imports.
https://www.python.org/dev/peps/pep-0008/#imports

Reply:
@g1itch pylint was complaining for 
Imports from package are not grouped
that' why i changed this.

Reply:
so your pylint is buggy

Reply:
utils should go at the end

Reply:
i think there should be a space here, maybe that's why it was complaining